src/gt_frontend/gtscript_to_gtir.py

```diff
@@ -0,0 +1,472 @@
+# -*- coding: utf-8 -*-
+#
+# Eve Toolchain - GT4Py Project - GridTools Framework
+#
+# Copyright (c) 2020, CSCS - Swiss National Supercomputing Center, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+import copy
+
+from gt_frontend.gtscript import Field, Local, Location, Mesh, TemporaryField
+from gt_frontend.gtscript_ast import (
+    Argument,
+    Assign,
+    BinaryOp,
+    Call,
+    Computation,
+    Constant,
+    Generator,
+    Interval,
+    IterationOrder,
+    LocationComprehension,
+    LocationSpecification,
+    Stencil,
+    SubscriptMultiple,
+    SubscriptSingle,
+    Symbol,
+)
+
+import eve
+import gtc.common as common
+import gtc.unstructured.gtir as gtir
+
+
+_reduction_mapping = {
+    "sum": gtir.ReduceOperator.ADD,
+    "product": gtir.ReduceOperator.MUL,
+    "min": gtir.ReduceOperator.MIN,
+    "max": gtir.ReduceOperator.MAX,
+}
+
+# Types
```

Comment:
remove?

Reply:
I moved and adopted the comment to the symbol table datastructure.